tools/generated_files/320/operatorsScore.csv

```diff
@@ -176,6 +176,7 @@ Md5,4
 MicrosToTimestamp,4
 MillisToTimestamp,4
 Min,4
+MinBy,4
```

Comment:
Amazed by a same score file generated for every shim layer.
According to my experience with P/Q Tools, I'd like to file a following up issue to update the score every time we add in a new expr.